### DISCUSSION TITLE
Can I use t or tc functions outside of Vue components?

### DISCUSSION THREAD

**Discussion by Pavel-Son** [intlify/vue-i18n#686] [Q&A]
Hi! I would like to use `t` in my translation service as I did on Vue 2 project using i18next.
So basically I need translate service inside of my Service worker and there is no need to import whole Vue there.
It would be perfect to just import `t` function

Thanks!

**Comment by kazupon** [comment]
You can use global scope.
I hope this issue will give you some ideas.
https://github.com/intlify/vue-i18n-next/issues/536#issuecomment-863020657
